frontend/src/components/Modals/Settings/index.jsx

```diff
@@ -132,6 +134,12 @@ function SettingTabs({ selectedTab, changeTab, settings, user }) {
           />
         </>
       )}
+      <SettingTab
+        displayName="Appearance"
+        tabName="appearance"
+        icon={<Eye className="h-4 w-4 flex-shrink-0" />}
+        onClick={() => window.open(paths.appearance())}
+      />
```

Comment:
This tab should only appear when *not* in multi use mode. Not show in all modes.